packages/@aws-cdk/aws-apigateway/test/integ.api-definition.inline.private.ts

```diff
@@ -0,0 +1,69 @@
+import * as cdk from '@aws-cdk/core';
```

Comment:
Instead of an integration test, add a unit test [here](https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-apigateway/test/test.restapi.ts#L998) instead.

Reply:
I added a unit test. Waiting for the build to confirm it works... my machine (and GitPot) unit test keep failing in jest timeouts so using this slower approach.

Reply:
Drop this integration test.